move/axelar_gateway/sources/gateway.move

```diff
@@ -72,7 +74,8 @@ public struct Gateway has key {
     signers: AxelarSigners,
 }
 
-// === Ticket ===
+/// [docstring]
+/// The version is captured to ensure that future packages can restrict which messages they can send, and to ensure that no future messages are sent from earlier versions.
 public struct MessageTicket {
```

Comment:
shall we rename this to just `Message`?
```suggestion
public struct Message {
```
we don't really suffix other hot potatoes